userspace/ksud/src/module.rs

```diff
@@ -570,6 +570,28 @@ pub fn uninstall_module(id: &str) -> Result<()> {
     })
 }
 
+pub fn run_action(id: &str) -> Result<()> {
```

Comment:
Why not use existing `exec_script`?

Reply:
I'm very sorry. I didn't notice that this function method exists. It has been optimized according to the situation.（qaq）